react/src/components/ManageMapProjectPanel/MapTabContent.tsx

```diff
@@ -0,0 +1,169 @@
+  project: Project;
+  onProjectUpdate: (updateData: Partial<ProjectRequest>) => void;
+  isPending: boolean;
+}
+
+const MapTabContent: React.FC<MapTabProps> = ({
+  project,
+  onProjectUpdate,
+  isPending,
+}) => {
+  const [editProjectField, setEditProjectField] = useState({});
+  const [isEditModalOpen, setisEditModalOpen] = useState(false);
+  const [currentField, setCurrentField] = useState<
+    'name' | 'description' | null
+  >(null);
+  const [editValue, setEditValue] = useState('');
+  const [validationError, setValidationError] = useState(false);
+
+  const handleEditClick = (fieldName: 'name' | 'description') => {
+    setCurrentField(fieldName);
+    setEditValue(project[fieldName]);
+    setisEditModalOpen(true);
+    setEditProjectField({
+      ...editProjectField,
+      [fieldName]: !editProjectField[fieldName],
+    });
+  };
+
+  const onEditSubmit = () => {
+    if (!currentField) return;
+
+    if (!editValue.trim()) {
+      setValidationError(true);
+      return;
+    }
+    setValidationError(false);
+
+    onProjectUpdate({ [currentField]: editValue });
+    setisEditModalOpen(false);
+    setCurrentField(null);
+    setEditProjectField({});
+  };
+
+  const handleEditModalCancel = () => {
+    setisEditModalOpen(false);
+    setValidationError(false);
+    setCurrentField(null);
+    setEditProjectField({});
+  };
+
+  //Deleting map
+  const [isDeleteModalOpen, setisDeleteModalOpen] = useState(false);
+  const handleDeleteClose = () => {
+    setisDeleteModalOpen(false);
+  };
+
+  const config = useAppConfiguration();
+
+  return (
+    <>
+      <Flex vertical justify="center">
+        <List itemLayout="vertical" style={{ marginLeft: 20, marginRight: 20 }}>
+          <List.Item>
+            <List.Item.Meta title={'Name:'} style={{ marginBottom: 0 }} />
+            <Flex justify="space-between">
+              {project.name}
+              <Button
+                onClick={() => handleEditClick('name')}
+                type="text"
+                icon={
+                  editProjectField['name'] ? <CheckOutlined /> : <EditFilled />
+                }
+              />
+            </Flex>
+          </List.Item>
+          <List.Item>
+            <List.Item.Meta
+              title={'Description:'}
+              style={{ marginBottom: 0 }}
+            />
+            <Flex justify="space-between">
+              {project.description}
+              <Button
+                onClick={() => handleEditClick('description')}
+                type="text"
+                icon={
+                  editProjectField['description'] ? (
+                    <CheckOutlined />
+                  ) : (
+                    <EditFilled />
+                  )
+                }
+              />
+            </Flex>
+          </List.Item>
+          <List.Item>
+            <Flex vertical justify="center" gap="small">
+              <Button
+                type="primary"
+                href={config.taggitUrl}
```

Comment:
This needs navigate to taggit but after setting a value in local storage.  This is our work-around approach as taggit doesn't have a url/route for individual map/galleries.

Can we create a JIra issue as a follow on and list it here (i.e. `// TODO Improve navtaging to taggit https://tacc-main.atlassian.net/browse/WG-XXX`)